CHALLENGE.md

```diff
@@ -52,14 +52,15 @@ You are given a prepared SPA project that was bootstrapped with Create React App
 ![edit-promp](.github/edit-prompt.png)
 
 - Clicking on the `EDIT` button will open a browser prompt with a message `New Tournament Name:`, an input field with the current tournament name as pre-filled value and the buttons `Cancel` to cancel and `OK` to confirm.
-- When confirming, the tournament name will be updated on the fake REST API and UI optimistically without any loading indicators.
+- The tournament name must contain only Latin letters, numbers, and spaces, not an empty string or only spaces.
+- When confirming, the tournament name will be updated immediately using an optimistic update in the UI and a fetch call on the fake REST API.
```

Comment:
Not sure what we said in our discussion. Shall we mention handling the error cases?

Reply:
I would mention it even if we didn't discuss it.

Reply:
as we didn't define how do we want to handle error cases, then I'll leave it like this for now
maybe it's a good question for the tech interview